packages/amplication-client/src/Resource/SyncWithGithubTile.tsx

```diff
@@ -30,9 +30,9 @@ function SyncWithGithubTile({ resourceId }: Props) {
 
   return (
```

Comment:
just a naming comment. git === github. if you want to make it generic you should address it as version control system or vcs. this will apply to all vcs like bitbucket, gitlab etc...

Reply:
I don't understand this comment